### ISSUE TITLE
List operations/facilities that the user has access to

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-reporting#123]
#### Description:

In order for an external user to file a report, they need to be able to tell us which operation they will be filing for, and we will need the list of facilities associated with the operations. Operations can consist of 1 (SFO) or more (LFO) facilities. We must only fetch the operations & facilities that the user has permission to see, which at least for now means they belong to the users operator. This information is entered in registration and we need to fetch it.

#### Acceptance Criteria:

Given I am logged in as an external user,
When I start to file a report,
Then I am presented with a list of my operations and their associated facilities

#### Development Checklist:

- [ ] Create a placeholder page to show the list 
- [ ] Add a link from the "Reporting logged-in homepage" to the new placeholder page
- [ ] Target the appropriate API endpoint from the registration API
- [ ] Display basic information on the placeholder page so that we can view the results
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By

This ticket is blocked by #163 

#### Notes:

-

**Comment by mikevespi** [comment]
I've left the visual representation quite vague for now. I interpreted the main goal of the card to be fetching the relevant data.

**Comment by pbastia** [comment]
I'll remove the blocking tag, this can live in a public space for now and be brought behind the login at a later point

**Comment by patriciarussellCAS** [comment]
@BallardRobinett how close are we to having this ticket complete. I see that it's blocking 217 so would be good to get this done quickly.
